conf/machine/hardkernel-odroidn2plus.conf

```diff
@@ -10,3 +10,7 @@ MACHINE_FEATURES_append = " alsa ext2 screen usbgadget usbhost vfat"
 
 
 UBOOT_MACHINE = "odroid-n2_defconfig"
+
+MACHINE_ESSENTIAL_EXTRA_RDEPENDS += " \
+       alsa-utils-alsactl \
```

Comment:
Are you sure this isn't selected with the `alsa` MACHINE_FEATURES ?

Reply:
Nope - will check that.